src/app/downloads/downloads.component.html

```diff
@@ -16,16 +16,15 @@ <h3 class="col-12 mb-2 mt-4 card-subtitle text-muted">Select your platform to st
             <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.windows_main" (click)="trackDownload('windows', latestRelease.windows_main)">Windows (x64)</a>
           </div>
           <div class="col-xs-12 col-sm-4 col-lg-4 mr-1">
-            <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.ubuntu_22" (click)="trackDownload('linux', latestRelease.ubuntu_22)">Ubuntu 22.04</a>
+            <a class="btn btn-default btn-primary btn-block btn-bg-dark mb-1" role="button" [href]="latestRelease.ubuntu_24" (click)="trackDownload('linux', latestRelease.ubuntu_24)">Ubuntu 24.04</a>
             <div class="alt-builds">
-              <a class="card-link text-center ml-2" [href]="latestRelease.ubuntu_20" (click)="trackDownload('linux', latestRelease.ubuntu_20)">Ubuntu 20.04</a>
+              <a class="card-link text-center ml-2" [href]="latestRelease.ubuntu_22" (click)="trackDownload('linux', latestRelease.ubuntu_22)">Ubuntu 22.04</a>
             </div>
           </div>
           <div class="col-xs-12 col-sm-2 col-lg-3">
-            <a class="btn btn-default btn-block btn-primary btn-bg-dark mb-1" role="button" [href]="latestRelease.mac12" (click)="trackDownload('mac', latestRelease.mac12)">MacOS 12</a>
+            <a class="btn btn-default btn-block btn-primary btn-bg-dark mb-1" role="button" [href]="latestRelease.mac13_arm" (click)="trackDownload('mac', latestRelease.mac13_arm)">MacOS 13 (ARM)</a>
```

Comment:
- ARM is on the mac13 image on GitHub for us, so updated that as the primary
- Mac x86 is on mac12
- Removed mac11